.changeset/modern-knives-bathe.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Console also need to be patched. Otherwise new release for console won't be created